list-pr-files/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Matheus Pimenta, Exercism
```

Comment:
Do we add Exercism or replace the original notice? I guess both work technically but which is nicer?

Reply:
🤷🏽‍♂️

Reply:
The (only!) conditions of the MIT license are

> **The above copyright notice** and this permission notice shall be included in all
> copies or substantial portions of the Software.

So I don't think we should remove Matheus Pimenta, even if we include the original license in `THIRD_PARTY_NOTICE.md`.

I believe it's common to write something like:
```
Copyright (c) 2020 Matheus Pimenta
Copyright (c) 2020 Exercism
```
But maybe the LICENSE file should be in the root of this repo, rather than on a per-subdirectory basis.

We have a LICENSE file here:
https://github.com/exercism/github-actions/blob/master/ensure-issue-authorization/LICENSE

but not in the `configlet-ci` and `canonical-data-syncer-ci` subdirectories.

Reply:
```suggestion 
Original work Copyright 2020 Matheus Pimenta 
Modified work Copyright 2020 Exercism  

```

Reply:
Does that still work with licensee etc?

> The (only!) conditions of the MIT license are

That condition is met, there's no requirement to keep the filename.